admin_guide/overcommit.adoc

```diff
@@ -74,7 +74,23 @@ filter-pre-mux-client.conf:
   buffer_path `/var/lib/fluentd/buffer-mux-client`
 ----
 
-The Fluentd `buffer_queue_limit` is 32.
+The Fluentd `buffer_queue_limit` is the value of the variable `BUFFER_QUEUE_LIMIT`. This value is `32` by default.
+
+The environment variable `BUFFER_QUEUE_LIMIT` is calculated as `(FILE_BUFFER_LIMIT / (number_of_outputs * BUFFER_SIZE_LIMIT))`.
+
+In this formula, `number_of_outputs` is `1` if all the logs are sent to a single resource and it is incremented by `1` for each additional resource. For example, the value of `number_of_outputs` will be:
+
+ * `1` - If all logs are sent to a single ElasticSearch node.
+ * `2` - Incremented by `1`, if ops ElasticSearch is additionally configured.
+ * `3` - Incremented by `1` again, if the logs are also forwarded to other Fluentd.
```

Comment:
This number_of_outputs is a bit tricky. :)  Not just the 1, 2, 3, but there could be more combinations.  
Single ElasticSearch for all -- 1 output
One ElasticSearch for ops and another for application logs -- 2 outputs
One ElasticSearch for all and forward them to other Fluentd -- 2 outputs
One ElasticSearch for ops and another for application logs plus forward one of the outputs to other Fluentd -- 3 outputs
One ElasticSearch for ops and another for application logs plus forward the both outputs to other Fluentd -- 4 outputs
Please note that I'm not suggesting to list up all of them, but I'd rather think `2` `3` is not needed.  Or do you have any good idea to explain this?  Thanks!